src/trio/_channel.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, max_buffer_size: int | float):  # noqa: PYI041
 
 
 @attrs.frozen
-class MemoryChannelStats:
+class MemoryChannelStatistics:
```

Comment:
I *think* this is backwards compatible as far as we care.

Reply:
If it wasn't publicaly re-exported originally, I would think this should count as backwards compatible, and if anyone does have issues we can tell them its their fault for digging into Trio's internals.